sparta/example/CoreModel/gen_layouts.py

```diff
@@ -0,0 +1,87 @@
+import sys,os
+
+sys.path.append('../../../../helios/pipeViewer/scripts')
```

Comment:
Is it possible to specify an absolute path here with a variable instead of the relative path?

Reply:
Yeah, I'll change it to build the abs path.

Reply:
Doine